src/ray/core_worker/store_provider/memory_store/memory_store.cc

```diff
@@ -198,8 +205,11 @@ Status CoreWorkerMemoryStore::Get(const std::vector<ObjectID> &object_ids,
       }
     }
 
-    for (const auto &object_id : ids_to_remove) {
-      objects_.erase(object_id);
+    // Clean up the objects if ref counting is off.
```

Comment:
Can you add a `TODO` somewhere about removing this logic once we have GC in Java.

Reply:
Done